### DISCUSSION TITLE
Having issue with kafka listener

### DISCUSSION THREAD

**Discussion by Lakshay1912** [strimzi/strimzi-kafka-operator#8541] [Q&A]
Hi @scholzj,

In my last deployment, I upgrade the kafka version 3.2.3,  confluent version 7.3.0, and spring kafka version 2.9.0 after onwards I am getting 502 bad gateway error in my application pods.

Is this the issue is because of these versions and any other reason it would be? 

Here I attached the microservice log which is having issue and my kafka broker pods logs. 

[transfer_service.zip](https://github.com/strimzi/strimzi-kafka-operator/files/11527443/transfer_service.zip)
[Archive.zip](https://github.com/strimzi/strimzi-kafka-operator/files/11527472/Archive.zip)

Please considered it as priority, it is a production issue. Thnx

**Comment by scholzj** [comment]
You should probably start by properly explaining the situation and your environment ... what are you using, how did you configured it, what are the steps you are doing, how are your clients configured etc. You shared the logs which is great, but without the custom resources and better understanding of what are you doing, they are not of a much value. It is also not clear how Confluent fits in as Strimzi has nothing to do with Confluent Platform.

**Reply by Lakshay1912** [reply]
basically, there is two springboot microservice in the system one is deployed on the azure using strimzi and aks. and other one is deployed on the company cloud which pull the records and send it over to the pod which is deployed on the azure. So in my last deployment I increased the above mentioned version after which I am getting the errors in the service which is deployed on-prem. and it is not able to build connection with the one which is already deployed on azure.

**Reply by Lakshay1912** [reply]
@scholzj

**Reply by scholzj** [reply]
Ok, let me put it differently. Where exactly you get the 502 error? From the log you seem to get it from some URL `https://api.ets.optum.com/bridge-service/generic-record/` => but that is not really a Strimzi URL in any way. So I do not understand how it is related.
